docs/discussions/process.md

```diff
@@ -0,0 +1,19 @@
+# Process #
+
+This doc describes some of our team's processes, ceremonies (a fancy term for meetings), and philosophy about working in an Agile way.
+
+## Predictability ##
+
+Our stakeholders want to see momentum and don't really care about how many story points we get done or what our velocity is, so we would like to find a way to reliably deliver something they care about (mainly components) every week.
+
+## Planning ##
+
+Planning is an agile ceremony where we decide what component we are going to publish in a given week and discuss the logistics necessary to make that happen. Addiitonally, we decide what long-running work is going to be happening in parallel and who is going to work on that.
+
+## Grooming ##
+
+Grooming is an agile ceremony where we make sure that we have a next priority big component and several next priority small components in the backlog.
+
+## Spikes ##
+
+Spikes are a term for a card on the board where the acceptance criteria doesn't have much if any customer value and instead serves to potentially de-risk future cards. We've decided as a team that we won't bother with spikes. Instead, we will just write a story to deliver value even if it's big and scary. "Bigness" and "scaryness" are better thought of as "risk" and every story has some amount of it. Every day, we have a stand up meeting where we're going to talk about our stories and if we decide it's not going fast enough, we can write up the investigation done so far, link it in the story and move the story back to ready or idea for a later time when we feel we can handle the risk, or can be broken down.
```

Comment:
Should we have a link directly to the Waffle board here since we're talking about the columns? I'd probably capitalize Ready and Idea here since we're talking about the names of the columns.